### PR TITLE
docs: adds readme banner

### PR DESCRIPTION
**Pull request by dvsrepo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
-<p align="left">
-    <img src="docs/images/rubrix_logo.svg" alt="drawing" width="200" />
-</p>
-<h3> Open-source framework to label, refine and monitor data for NLP</h3>
 
-https://user-images.githubusercontent.com/1107111/148930791-ed4c3640-aa74-41b1-bfc5-5717e137bc1b.mp4
+<div align="center">
+    <h2><a href="https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g"> 👩🏾‍💻 Join the community on Slack</a>
+    | <a href="https://docs.rubrix.ml">📚 Docs</a>
+| <a href="https://github.com/recognai/rubrix/#get-started">🚀 Get started</a><
```
Comment (dvsrepo) [inline README.md]:
```suggestion
| <a href="https://github.com/recognai/rubrix/#get-started">🚀 Get started</a>
```

File: README.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
-<p align="left">
-    <img src="docs/images/rubrix_logo.svg" alt="drawing" width="200" />
-</p>
-<h3> Open-source framework to label, refine and monitor data for NLP</h3>
 
-https://user-images.githubusercontent.com/1107111/148930791-ed4c3640-aa74-41b1-bfc5-5717e137bc1b.mp4
+<div align="center">
+    <h2><a href="https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g"> 👩🏾‍💻 Join the community on Slack</a>
```
Comment (dvsrepo) [inline README.md]:
```suggestion
    <h3><a href="https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g"> 👩🏾‍💻 Join the community on Slack</a>
```

File: README.md [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+    | <a href="https://docs.rubrix.ml">📚 Docs</a>
+| <a href="https://github.com/recognai/rubrix/#get-started">🚀 Get started</a><
+    | <a href="https://github.com/recognai/rubrix/#quick-links">🔗 Quick links</a></h2>
```
Comment (dvsrepo) [inline README.md]:
```suggestion
    | <a href="https://github.com/recognai/rubrix/#quick-links">🔗 Quick links</a></h3>
```

**Comment by dvsrepo** [comment]
@dcfidalgo sorry David, could you review this one, I hadn't seen your other PR.

**Review by dcfidalgo — APPROVED** [review]

**Pull request merged**